test/e2e/app-dir/io-is-dynamic-ppr/dynamic-io-ppr.test.ts

```diff
@@ -0,0 +1,281 @@
+import { nextTestSetup } from 'e2e-utils'
+
+process.env.__TEST_SENTINEL = 'at buildtime'
+
+describe('dynamic-io', () => {
+  const { next, isNextDev, skipped } = nextTestSetup({
+    files: __dirname,
+    skipStart: true,
+    skipDeployment: true,
+  })
+
+  if (skipped) {
+    return
+  }
+
+  beforeAll(async () => {
+    await next.start()
+    await next.render('/bootstrap_runtime')
+  })
```

Comment:
`nextTestSetup` I believe calls `start()` so it looks like we're building + starting & then stopping and starting again. Do we need to do that?